checks.d/jenkins.py

```diff
@@ -137,6 +137,16 @@ def _get_build_results(self, instance_key, job_dir):
             self.log.error("Error while working on job %s, exception: %s" % (job_name, e))
 
     def check(self, instance, create_event=True):
+        """
+        DEPRECATED:
+        This Jenkins check is deprecated and not actively developed anymore. It will be
+        removed in a future version of the Datadog Agent. Please move to using the Datadog
+        plugin for Jenkins. More information can be found un the Jenkins Integration panel
+        under the Configuration tab (https://app.datadoghq.com/account/settings#integrations/jenkins)
```

Comment:
Jenkins tile on the integration panel does not mention the Datadog plugin for Jenkins yet. Is there a PR coming soon ?